pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/slack/index.md

```diff
@@ -0,0 +1,129 @@
+  # alerts as-is. This is unlikely to be what you want, unless you have
+  # a very low alert volume or your upstream notification system performs
+  # its own grouping. Example: group_by: [...]
+  group_by: ['alertname', 'cluster', 'service']
+
+  # When a new group of alerts is created by an incoming alert, wait at
+  # least 'group_wait' to send the initial notification.
+  # This way ensures that you get multiple alerts for the same group that start
+  # firing shortly after another are batched together on the first
+  # notification.
+  group_wait: 30s
+
+  # When the first notification was sent, wait 'group_interval' to send a batch
+  # of new alerts that started firing for that group.
+  group_interval: 5m
+
+  # If an alert has successfully been sent, wait 'repeat_interval' to
+  # resend them.
+  repeat_interval: 30m
+
+  # A default receiver
+  receiver: slack_general
+
+receivers:
+- name: slack_general
+  slack_configs:
+  - channel: '#prometheus-alerts'
+```
+
+Save `config.yml` into a Git repository, in a folder called `slack`.
+Assume that the repository is `https://github.com/company/alertmanager-configs`.
+
+## Create the Slack API URL secret
+
+The service is automatically configured to add the following global default value for `slack_api_url` in the configuration file.
+
+```yaml
+global:
+- slack_api_url: '<SLACK_API_URL>'
+```
+
+Create the secret `slackapiurl-secret` for the Slack API URL ([Webhook URL](https://api.slack.com/incoming-webhooks)) to use for notifications.
+
+```bash
+dcos security secrets create --value=<SLACK_API_URL> slackapiurl-secret
+```
+
+When installing the service, you must configure the `secrets` section to point to the secret `slackapiurl-secret` created.
+
+```json
+{
+  "alertmanager": {
+    "secrets": {
+      "slack_api_url_secret": "slackapiurl-secret"
+    },
+    "config_repository": {
+      "url": "https://github.com/company/alertmanager-configs",
+      "path": "/slack"
+    }
+  }
+}
+```
+
+If the Git repository is private, you will need to [configure the credentials](../config/#create-secrets-for-git-repository-credentials) to access the Git repository.
+
+## Install {{ model.techName }} service
+
+Install the service using the custom options file (`options.json`) created in the above steps:
+
+```bash
+dcos package install {{ model.serviceName }} --options=options.json
+```
+
+The Alertmanager configurations defined in the repository will be automatically loaded when the service finishes deploying.
+
+## Triggering a reload of Alertmanager configurations
+
+It's possible to trigger a reload of the Alertmanager configurations after the service is installed.
+
+```bash
+dcos monitoring plan start reload-alertmanager-config
+```
+
+## Testing Slack alerts
+
+You can test that Alertmanager is configured correctly by SSH to the node on which Alertmanager is running and manually hitting the Alertmanager endpoint to trigger an alert.
```

Comment:
"correctly by SSH to the node" is awkward.

how about:

"configured correctly by using SSH to connect to the node"